config/config_test.go

```diff
@@ -100,12 +113,12 @@ var expectedConf = &Config{
 			MetricsPath: DefaultScrapeConfig.MetricsPath,
 			Scheme:      DefaultScrapeConfig.Scheme,
 
-			HTTPClientConfig: HTTPClientConfig{
+			HTTPClientConfig: config_util.HTTPClientConfig{
 				BearerTokenFile: filepath.FromSlash("testdata/valid_token_file"),
 			},
 
-			ServiceDiscoveryConfig: ServiceDiscoveryConfig{
-				StaticConfigs: []*TargetGroup{
+			ServiceDiscoveryConfig: sd_config.ServiceDiscoveryConfig{
```

Comment:
`sd_config.ServiceDiscoveryConfig` stutters now, but I think that's ok to fix in a later change that removes `config` deps from the main `discovery` package and thus allows us to just move the main SD config there.